authors/apps/articles/admin.py

```diff
@@ -0,0 +1,4 @@
+from django.contrib import admin
+from .models import ArticlesModel
+# Register your models here.
+
```

Comment:
Hello, could we register the `ArticlesModel` with the admin as well?

Reply:
Done

Reply:
Cool.